libpod/options.go

```diff
@@ -192,6 +192,24 @@ func WithConmonEnv(environment []string) RuntimeOption {
 	}
 }
 
+// WithNetworkCmdPath specifies the path to the slirp4netns binary which manages the
+// runtime.
+func WithNetworkCmdPath(path string) RuntimeOption {
+	return func(rt *Runtime) error {
+		if rt.valid {
+			return ErrRuntimeFinalized
+		}
+
+		if path == "" {
+			return errors.Wrapf(ErrInvalidArg, "must provide a valid path")
```

Comment:
Since we allow `""` above (to indicate we just use `$PATH`), should we allow the empty string here and document that it indicates we will look it up using the system's `$PATH`?